### ISSUE TITLE
Line feeds (or carriage returns) being deleted

### ISSUE THREAD

**Issue by jcamp** [penge/my-notes#425]
When you copy to a note from the website the line endings disappear. Would be handy to have an option to either keep line endings or surround the markdown with ``` xxx ``` to keep?

**Comment by penge** [comment]
Hi @jcamp

Can you provide some example that I could follow to reproduce this problem?

**Comment by jcamp** [comment]
Yes, just right click any highlighted text from a web page and add to a note. Formatting and CR's are stipped which makes reading it very difficult.

What the highlighted web content looks like:
![Image](https://github.com/user-attachments/assets/1dafb9da-3b17-4f53-98dd-8c5e47addb57)

What the "My Notes" looks like:
![Image](https://github.com/user-attachments/assets/c4a03281-1fdd-415d-b009-2e1eaec0ef31)
